stories/button/button.stories.js

```diff
@@ -8,9 +8,25 @@ import '../../dist/segmented-button.css';
 export default {
     title: 'Components/Button',
     parameters: {
-        description: `Buttons allow users to perform actions.
-        All the buttons require the fd-button base class and an additional modifier
-        for each of the types.`
+        description: `The button component is used to trigger an action. All buttons are characterized by the <code class="docs-code">fd-button</code> class and an additional modifier class to specify each button type.
+
+##Usage
+
+**Use the button types as follows:**
+
+- Use simple buttons for specific actions, such as:
+    - _Create_, _Edit_, _Save_
+    - _Approve_, _Reject_
+    - _Accept_, _Decline_
+    - _OK_, _Cancel_
+- If you want the user to select one option from a small group, use a segmented button. For example: _Small_, _Medium_, _Large_
+- Use the menu button if you need a menu that provides more than one option.
+
+**Do not use buttons if:**
+
+- You want to link to a different page or object. Instead, use the **Link** component.
+- You want to let users upload content. Instead, use the **Upload Collection** component.
```

Comment:
we need to be very careful here - we don't have `Upload Collection` (yet)

Reply:
True. I'll remove that.